src/openforms/appointments/contrib/jcc/tests/test_plugin.py

```diff
@@ -0,0 +1,144 @@
+import os
+from datetime import date, datetime
+
+from django.test import TestCase
+
+import requests_mock
+from zeep.client import Client
+
+from openforms.appointments.contrib.base import (
+    AppointmentClient,
+    AppointmentLocation,
+    AppointmentProduct,
+)
+
+from ..plugin import Plugin
+
+
+def mock_response(filename):
+    filepath = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), "mock", filename)
+    )
+    with open(filepath, "r") as f:
+        return f.read()
+
+
+class PluginTests(TestCase):
+    maxDiff = 1024
+
+    def setUp(self, *args, **kwargs):
+        self.plugin = Plugin()
+
+        wsdl = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), "mock/GenericGuidanceSystem2.wsdl")
+        )
+        self.plugin.client = Client(wsdl)
+
+    def test_get_available_products(self):
+        with requests_mock.mock() as m:
```

Comment:
```suggestion
    @requests_mock.Mocker()
    def test_get_available_products(self, m):
```

the decorator approach saves you a level of indentation